pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -257,16 +260,16 @@ func TestUserSignupWithManualApprovalNotApproved(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, reconcile.Result{}, res)
 
+	// Lookup the userSignup again
+	err = r.client.Get(context.TODO(), types.NamespacedName{Name: userSignup.Name, Namespace: req.Namespace}, userSignup)
+	require.NoError(t, err)
+
 	mur := &v1alpha1.MasterUserRecord{}
 	err = r.client.Get(context.TODO(), types.NamespacedName{Name: userSignup.Name, Namespace: req.Namespace}, mur)
```

Comment:
I don't think that the MUR should be found using `Name: userSignup.Name`

Reply:
Good catch, fixed.